ert_shared/main.py

```diff
@@ -441,6 +441,12 @@ def start_ert_server(mode: str):
         yield
 
 
+def log_config(args: ArgumentParser, logger: logging.Logger) -> None:
+    with open(args.config, "r", encoding="utf-8") as file_obj:
+        content = file_obj.read()
+        logger.info(f"Content of the configuration file ({args.config}):\n" + content)
```

Comment:
would it make sense to call this `debug` logging? (not having checked if that will exclude if from being picked up where we want it later)

Reply:
I was also considering that, but maybe it is also useful for the case you change your config and then want to check the logs to see what was actually used before.

Reply:
Actually, that is a bit far-fetched, I will make it `debug`.